frontend/src/components/GoalCards/GoalCard.js

```diff
@@ -1,206 +1,206 @@
-            regionId={regionId}
-          />
-        </div>
-        { !hideGoalOptions && (
-        <ContextMenu
-          label={contextMenuLabel}
-          menuItems={menuItems}
-        />
-        )}
-      </div>
-      <div className={`display-flex flex-wrap margin-y-2 ${internalLeftMargin}`}>
-        <div className="ttahub-goal-card__goal-column ttahub-goal-card__goal-column__goal-text padding-right-3">
-          <h3 className="usa-prose usa-prose margin-y-0">
-            Goal
-            {' '}
-            {goalNumbers}
-          </h3>
-          <p className="text-wrap usa-prose margin-y-0">
-            {goalText}
-            {' '}
-            {determineFlagStatus()}
-          </p>
-        </div>
-        <div className="ttahub-goal-card__goal-column ttahub-goal-card__goal-column__goal-topics padding-right-3">
-          <p className="usa-prose text-bold margin-y-0">Topics</p>
-          <Topics topics={goalTopics} />
-        </div>
-        <div className="ttahub-goal-card__goal-column ttahub-goal-card__goal-column__created-on padding-right-3">
-          <p className="usa-prose text-bold  margin-y-0">Created on</p>
-          <p className="usa-prose margin-y-0">{moment(createdOn, 'YYYY-MM-DD').format(DATE_DISPLAY_FORMAT)}</p>
-        </div>
-        <div className="ttahub-goal-card__goal-column ttahub-goal-card__goal-column__last-tta padding-right-3">
-          <p className="usa-prose text-bold margin-y-0">Last TTA</p>
-          <p className="usa-prose margin-y-0">{lastTTA}</p>
-        </div>
-        <div className="ttahub-goal-card__goal-column ttahub-goal-card__goal-column__last-reviewed padding-right-3">
-          <p className="usa-prose text-bold margin-y-0">Last reviewed</p>
-        </div>
-      </div>
-
-      <div className={internalLeftMargin}>
-        <ObjectiveButton
-          closeOrOpenObjectives={closeOrOpenObjectives}
-          objectiveCount={objectiveCount}
-          objectivesExpanded={objectivesExpanded}
-          goalNumber={goal.goalNumbers.join('')}
-        />
-      </div>
-      {objectives.map((obj) => (
-        <ObjectiveCard
-          key={`objective_${uuidv4()}`}
-          objective={obj}
-          objectivesExpanded={objectivesExpanded}
-        />
-      ))}
-
-    </article>
-  );
-}
-
-GoalCard.propTypes = {
-  goal: goalPropTypes.isRequired,
-  recipientId: PropTypes.string.isRequired,
-  regionId: PropTypes.string.isRequired,
-  showCloseSuspendGoalModal: PropTypes.func.isRequired,
-  performGoalStatusUpdate: PropTypes.func.isRequired,
-  handleGoalCheckboxSelect: PropTypes.func.isRequired,
-  isChecked: PropTypes.bool.isRequired,
-  hideCheckbox: PropTypes.bool,
-  showReadOnlyStatus: PropTypes.bool,
-  hideGoalOptions: PropTypes.bool,
-  erroneouslySelected: PropTypes.bool,
-};
-
-GoalCard.defaultProps = {
-  hideCheckbox: false,
-  showReadOnlyStatus: false,
-  hideGoalOptions: false,
-  erroneouslySelected: false,
-};
-
-export default GoalCard;
```

Comment:
did something change with your local config? seems like files are getting line endings switched, lol... hard to tell what's changed in these cases

Reply:
Yea I switched it to what I thought everyone else was on but maybe its wrong. 

[core]
	autocrlf = false